ambry-named-mysql/src/test/java/com/github/ambry/named/MySqlNamedBlobDbTest.java

```diff
@@ -133,6 +133,14 @@ public void testQueryExecutionFailure() throws Exception {
         e -> Assert.assertEquals(sqlException, e.getCause()));
   }
 
+  @Test
+  public void testPullAndCleanStaleNamedBlobs() throws Exception {
+    dataSourceFactory.setLocalDatacenter(localDatacenter);
+    dataSourceFactory.triggerEmptyResultSetForLocalDataCenter(datacenters);
+    List<StaleNamedResult> staleNamedResults = namedBlobDb.pullStaleBlobIds().get();
+    namedBlobDb.cleanupStaleData(staleNamedResults);
+  }
```

Comment:
this test is weird, it doesn't check the state of database. Can you write a bunch of stale data to the database and then verify pullStaleBlodIds get those data out. And can you also verify that after cleanupStateData, pullStaleBlobIds won't pull them out again.

Reply:
Added the proper tests in MySqlNamedBlobDbIntegrationTest in latest commit [1594fb9](https://github.com/linkedin/ambry/pull/2327/commits/1594fb9f2aa93ecac7c1d4f7a2b3109d196ec289)